control-plane/drizzle/0019_early_silver_sable.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE "users";--> statement-breakpoint
+ALTER TABLE "jobs" ADD COLUMN "resulted_at" timestamp with time zone;--> statement-breakpoint
+ALTER TABLE "jobs" ADD COLUMN "function_execution_time_ms" integer;--> statement-breakpoint
+ALTER TABLE "jobs" ADD COLUMN "service" varchar(1024);
```

Comment:
```suggestion
ALTER TABLE "jobs" ADD COLUMN IF NOT EXISTS "service" varchar(1024);
```

This was created, but not committed as a part of a previous deployment. This should fix your build.

Reply:
Nice, thanks!